src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -620,6 +692,31 @@ public static string TestInvalidMethod(JToken test)
 
         public new string RedeclaredBaseMethod() => "child";
 
+        [MethodName("child/InvokeVirtualMethodOverride")]
+        public override string InvokeVirtualMethodOverride() => $"child {nameof(InvokeVirtualMethodOverride)}";
+
+        [MethodName("test/InvokeTestMethod")]
+        public string InvokeTestMethodAttribute() => "test method attribute";
+
+        public override string InvokeVirtualMethodNoOverride() => $"child {nameof(InvokeVirtualMethodNoOverride)}";
```

Comment:
From a usability standpoint, I'm having a hard time deciding whether I would predict the `[MethodName]` attribute to be inherited from the base class or not. whichever way you're going, can you document it on the attribute class?